documentation/modules/exploit/unix/webapp/trixbox_ce_endpoint_devicemap_rce.md

```diff
@@ -0,0 +1,90 @@
+## Introduction
+
+This module exploits a post-authentication OS command injection vulnerability found in Trixbox CE <= v2.8.0.4 in the "network" POST parameter of the "/maint/modules/endpointcfg/endpoint_devicemap.php" page. Successful exploitation allows for arbitrary command execution on the underlying operating system as the "asterisk" user.
+
+### Installation And Setup
+
+Trixbox CE v2.8.0.4 Inatallation: 
```

Comment:
Inatallation <- Misspelling, change to "Installation"

Reply:
Oops. Fixed

Reply:
Confirmed fixed.